### PR TITLE
made table to look like addressbook image

### PR DESCRIPTION
**Pull request by hennigk**

### CODE REVIEW

File: exercise-8.txt [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+mysql> DESCRIBE Account;+------------+--------------+------+-----+---------+----------------+
+| Field      | Type         | Null | Key | Default | Extra          |
++------------+--------------+------+-----+---------+----------------+
+| id         | int(11)      | NO   | PRI | NULL    | auto_increment |
+| email      | varchar(255) | NO   |     | NULL    |                |
+| password   | varchar(40)  | NO   |     | NULL    |                |
+| createdOn  | datetime     | NO   |     | NULL    |                |
+| modifiedON | datetime     | YES  |     | NULL    |                |
++------------+--------------+------+-----+---------+----------------+
+ALTER TABLE AddressBook ADD FOREIGN KEY (accountId) REFERENCES Account(id);
```
Comment (nyanofthemoon) [inline exercise-8.txt]: :trophy:

File: exercise-8.txt [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+| type             | enum('home','work','other')     | NO   |     | NULL    |                |
+| subtype          | enum('phone','address','email') | NO   |     | NULL    |                |
+| contentLineOne   | varchar(255)                    | NO   |     | NULL    |                |
+| contentLineTWO   | varchar(255)                    | YES  |     | NULL    |                |
+| contentLineThree | varchar(255)                    | YES  |     | NULL    |                |
+| contentLineFour  | varchar(255)                    | YES  |     | NULL    |                |
+| contentLineFive  | varchar(255)                    | YES  |     | NULL    |                |
++------------------+---------------------------------+------+-----+---------+----------------+
+
+ALTER TABLE Entry ADD FOREIGN KEY (addressBookId) REFERENCES AddressBook(id);
```
Comment (nyanofthemoon) [inline exercise-8.txt]: :trophy:

File: exercise-8.txt [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+mysql> DESCRIBE Entry;
++------------------+---------------------------------+------+-----+---------+----------------+
+| Field            | Type                            | Null | Key | Default | Extra          |
++------------------+---------------------------------+------+-----+---------+----------------+
+| id               | int(11)                         | NO   | PRI | NULL    | auto_increment |
+| addressBookID    | int(11)                         | NO   | MUL | NULL    |                |
```
Comment (nyanofthemoon) [inline exercise-8.txt]: Please use the correct column name defined in the specification.

File: exercise-8.txt [hunk 4]
```diff
@@ -0,0 +1,199 @@
...
+ALTER TABLE Entry ADD COLUMN contentLineThree VARCHAR(255);
+ALTER TABLE Entry ADD COLUMN contentLineFour VARCHAR(255);
+ALTER TABLE Entry ADD COLUMN contentLineFive VARCHAR(255);
+
+mysql> DESCRIBE Entry;
++------------------+---------------------------------+------+-----+---------+----------------+
+| Field            | Type                            | Null | Key | Default | Extra          |
++------------------+---------------------------------+------+-----+---------+----------------+
+| id               | int(11)                         | NO   | PRI | NULL    | auto_increment |
+| addressBookID    | int(11)                         | NO   |     | NULL    |                |
```
Comment (nyanofthemoon) [inline exercise-8.txt]: Please use the correct column name defined in the specification.

File: exercise-8.txt (hunk 3 above)
Comment (hennigk) [inline exercise-8.txt]: Ok. I made the changes. this was the command I used: 
ALTER TABLE Entry CHANGE contentLineTWO contentLineTwo varchar(255);

**Pull request closed without merging**